app/styles/components/_ratings.scss

```diff
@@ -198,11 +198,6 @@
   }
 }
 
-// Hover element (rating-button)
-.rating-button-dropdown {
-  margin-bottom: 10px;
-}
-
```

Comment:
This change breaks the rating widget on other parts of the site. This needs to be reversed and instead remove the margin on the library page only.